index.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const defaultOptions = {
+    styles: '*:focus  { outline: none !important; }',
+    styleSheetParent: document.head,
+};
+
+const createStylesheet = (options) => {
+    const style = document.createElement('style');
+    const targetNode = options.styleSheetParent;
+
+    style.innerText = options.styles;
+
+    return {
+        update: () => {
+            targetNode.append(style);
+        },
+        destroy: () => {
+            if (style.parentNode === targetNode) {
+                targetNode.removeChild(style);
+            }
+        },
+    };
+};
+
+const createListeners = (styleSheet) => {
+    let outlinesEnabled = true;
+
+    const handleKeydown = (ev) => {
+        const acceptedKeyCodes = [9, 37, 38, 39, 40];
```

Comment:
What do you think if we create an object, possibly at the top of the file, to establish a map between the key name and its code? Otherwise, I find this approach harder to read and maintain.

Reply:
Sounds good 👍